src/fpp.h

```diff
@@ -9,27 +9,28 @@ namespace tricycle {
 
 /// @class fpp
 ///
-/// This Facility is intended
-/// as a skeleton to guide the implementation of new Facility
-/// agents.
-/// The fpp class inherits from the Facility class and is
+/// The Reactor class inherits from the Facility class and is
```

Comment:
But this is the FPP class, so shouldn't it still refer to `fpp`?

Now that I've approved the previous PR, I'm wondering if we actually want to use the full name "FusionPowerPlant"?

Reply:
This was my fault for just copying things over from the descriptions on the old version of Tricycle, and yeah, I think FusionPowerPlant is maybe better (more descriptive). This can be fixed pretty easy by just redoing cycstub and then adding the resbufs back in. Good catch.